distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2301,6 +2301,18 @@ Event event = new Event("ddf/notification/catalog/downloads", properties);
 eventAdmin.postEvent(event);
 ----
 
```

Comment:
Looks like all of the other configurations are more oriented towards a specific area of the system or topics rather than config files. For instance, I could see this information being part of a _System Performance and Tuning_ section or something. Did you check with @ricklarsen to see if this was the best place to put this?